next/components/sections/WelcomeSection.tsx

```diff
@@ -49,7 +52,23 @@ export const WelcomeSection = ({ mainMenuItems, homepageHeader }: Props) => {
           />
         </div>
       </div>
-      <HomepageMenu items={mainMenuItems} />
+      <div ref={elementRef} className="w-max h-2 lg:block hidden" />
```

Comment:
I would move this below the welcome menu so sticky menu will appear little bit later.

Reply:
Plus there is no welcome menu on tables size screen -> change `lg:block` to `md:block` should fix it.